bigquery_etl/backfill/utils.py

```diff
@@ -262,12 +263,14 @@ def _should_complete(
         qualified_table_name, backfill.entry_date
     )
     if not _table_exists(client, staging_table):
+        click.echo(f"Backfill staging table does not exist: {qualified_table_name}")
```

Comment:
Should this be the staging table?
```suggestion
        click.echo(f"Backfill staging table does not exist: {staging_table}")
```